Dockerfile

```diff
@@ -16,5 +12,5 @@ WORKDIR /dotnetapp/HiP-UserStore
 EXPOSE 5000
 
 RUN dotnet restore --no-cache
-
+RUN chmod +x /dotnetapp/HiP-Achievements/run.sh
```

Comment:
It should be HiP-UserStore instead of HiP-Achievements

Reply:
🤦‍♂️